docs/api-graduation-guidelines.md

```diff
@@ -0,0 +1,31 @@
+This document defines the expectations of how API graduation is handled across the github.com/kubevirt organization. Individual projects within the KubeVirt org may decide to append additional criteria and timeline constraints to the items outlined here, but should not deviate from the broad expectations as defined in this document.
+
+# Graduation Phase Expecations
+
+## Alpha
+* Feature incomplete, meaning the API might be half implemented as functionality lands in phases.
+* Basic testing exists proving that the merged functionality works e2e.
+* No backwards compatibility guarantees meaning API can mutate and be deprecated without warning.
```

Comment:
"Be deprecated without warning" - as I see it, the deprecation _is_ the warning. In the case of an alpha API, there's no guarantee for a deprecation period before removal - things can just disappear. Clearly we _may_ deprecate/warn, but again - no guarantees.

Reply:
so maybe say "without further warning"?

Reply:
My point was, alpha APIs don't guarantee a deprecation period.
So: "Can mutate and be removed without (further) warning" or "without being deprecated".

Reply:
i changed this to replace the word `deprecated` to the api can be `removed` without warning.